.releaserc

```diff
@@ -0,0 +1,146 @@
+{
+  "branches": [
+    "master",
+    {
+      "name": "next",
+      "prerelease": true
+    }
+  ],
```

Comment:
We should probably remove the prerelease channel, since we're only running the release workflow on master anyways...

```suggestion
  "branches": [
    "master"
  ],
```